meta-mel/recipes-support/consolekit/consolekit/console-kit-shutdown-handler

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
@kergoth, is it a good idea to use bash explicitly or should we lean towards /bin/sh?

Reply:
Generally #!/bin/sh is preferred unless bashisms are truly required. If the recipe already depends/rdepends on bash it'd be less of a concern, but we don't want to pull bash into a build if we don't actually need it.

Reply:
@bboccoqq, what do you think we should do here? Does your script have any bashisms?

Reply:
I always write scripts using bash, so I wasn't sure if I had bashisms in there, but I ran it through a checker it complained about the use of -o in two tests at the end.

Reworked those lines, invokes #!/bin/sh, remove RDEPENDS_${PN] = "bash".